ionic_user_interface/src/views/New.vue

```diff
@@ -16,15 +16,23 @@
           :width="canvasWidth"
           :ionContent="ionContent"
         />
+        <div v-if="!photoUploaded">
+          <br />
+          <ion-button @click="handleClickUserGuide" color="medium" fill="clear">
```

Comment:
Idk if it's just me but I feel like adding this button makes the page look off-balanced (not horizontally centered) or smth

Reply:
Removed the icon to center the text